conductor_api/src/interface.rs

```diff
@@ -552,13 +552,14 @@ impl ConductorApiBuilder {
             let name = Self::get_as_string("name", &params_map)?;
             let public_address = Self::get_as_string("public_address", &params_map)?;
             let key_file = Self::get_as_string("key_file", &params_map)?;
+            let holo_remote_key = Self::get_as_bool("key_file", &params_map).unwrap_or_default();
```

Comment:
This is the only significant change, the rest is refactoring to remove the Option